addons/mail/static/src/messaging/component/composer/composer.xml

```diff
@@ -59,7 +59,7 @@
                             'o-composer-is-compact': props.isCompact,
                             'o_Composer_textInput-mobile': env.entities.Device.instance.isMobile,
                         }"
-                        composer="props.composer"
+                        composerLocalId="props.composerLocalId"
```

Comment:
props in template

Reply:
can just be composer.localId, since composer is set when evaluating this part of the template